### PR TITLE
[FEATURE] MS-27: Admins with Redux

### PR DESCRIPTION
**Pull request by juanandres7698**
Using Redux for Admin Site - CRUD

### CODE REVIEW

File: src/redux/admins/actions.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  return {
+    type: GET_ADMINS_FULFILLED,
+    payload
+  };
+};
+
+export const getAdminsRejected = (payload) => {
+  return {
+    type: GET_ADMINS_REJECTED,
+    payload
```
Comment (Sabrucita) [inline src/redux/admins/actions.js:39]: If I'm not wrong it's not necessary the payload in all de rejected options

File: src/redux/reducers.js [hunk 2]
```diff
@@ -1,8 +1,12 @@
 import applicationReducer from './applications/reducer';
+import adminsReducer from './admins/reducer';
+import modalReducer from './modal/reducer';
 import { combineReducers } from 'redux';
 
 const reducer = combineReducers({
-  application: applicationReducer
```
Comment (Sabrucita) [inline src/redux/reducers.js:5]: When you resolve conflicts don't forget to choose the application with S like this:
applications: applicationReducer,

**Review by Sabrucita — APPROVED** [review]

**Review by JamirMansilla — APPROVED** [review]
Approved 🏎️ 🌫️

File: src/constants/index.js [hunk 3]
```diff
@@ -2,6 +2,29 @@
...
+export const UPDATE_ADMIN_REJECTED = 'UPDATE_ADMIN_REJECTED';
+
+export const DELETE_ADMIN_FETCHING = 'DELETE_ADMIN_FETCHING';
+export const DELETE_ADMIN_FULFILLED = 'DELETE_ADMIN_FULFILLED';
+export const DELETE_ADMIN_REJECTED = 'DELETE_ADMIN_REJECTED';
+
+export const UPDATE_SELECTED_ADMIN = 'UPDATE_SELECTED_ADMIN';
+
+export const ADMINS_CLEANUP = 'ADMINS_CLEANUP';
 export const GET_APPLICATION_FETCHING = 'GET_APPLICATION_FETCHING';
```
Comment (BaltazarCortina) [inline src/constants/index.js:28]: I think here you deleted the "// APPLICATIONS" line, that separates one resource from the other

**Pull request merged**